ui/webui/src/components/localization/InstallationLanguage.jsx

```diff
@@ -317,9 +318,16 @@ LanguageSelector.contextType = AddressContext;
 export const InstallationLanguage = ({ idPrefix, setIsFormValid, onAddErrorNotification }) => {
     const [nativeName, setNativeName] = React.useState(false);
     const { setLanguage } = React.useContext(LanguageContext);
+    const [distributionName, setIsDistributionName] = useState("");
```

Comment:
the setter name needs fix. s/setIsDistributionName/setDistributionName
(remove the Is)

Reply:
It's common to use 'Is' as part of the state var if it's boolean. For example:
const [isValid, setIsValid] = useState(true);